app/src/main/java/skills/future/planer/MainActivity.java

```diff
@@ -48,12 +52,18 @@ private void displayVersionOfApplication(Menu menu) {
         }
     }
 
+    @SneakyThrows
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         themePreferences();
 
+        startService(new Intent(this, NotificationService.class));
```

Comment:
To przy każdym uruchomieniu uruchamia ten serwis czy jak on już jest to nie?

Reply:
to tak nie do końca działa